geniza/corpus/tests/test_corpus_admin.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from django.contrib import admin
+
+from geniza.corpus.models import LanguageScript, Document
+from geniza.corpus.admin import LanguageScriptAdmin
+
+class TestLanguageScriptAdmin(TestCase):
+    def test_borrows(self):
```

Comment:
test borrows?

Reply:
whoops, copied and pasted from s+co. Good catch! I've addressed everything else. @rebeccakoeser please confirm that I should merge